openshift-knative-operator/pkg/serving/extension.go

```diff
@@ -88,6 +93,11 @@ func (e *extension) Reconcile(ctx context.Context, comp v1alpha1.KComponent) err
 		}
 	}
 
+	logger := logging.FromContext(ctx)
+	logger.Info("Installing Serving Monitoring Requirements")
```

Comment:
Should we remove this logline? If monitoring is disabled, we won't do anything here.